spec/features/items/show.html.erb_spec.rb

```diff
@@ -8,12 +8,14 @@
     item = create(:item, owning_user: owner)
     item.waitlist = create(:waitlist_with_item)
     item.waitlist.item = item
+    item.users_with_direct_lend_permission << user
     item
   end
   let(:item_lent) do
     item_lent = create(:lent, owning_user: owner, holder: borrower.id)
     item_lent.waitlist = create(:waitlist_with_item)
     item_lent.waitlist.item = item_lent
+    item_lent.users_with_direct_lend_permission << user
```

Comment:
Although the behavior should:tm: not change, I'd rather use groups for the permissions here as the frontend only allows assigning lend and see permissions to groups at the moment